lib/nerves_network/eapol_manager.ex

```diff
@@ -476,6 +502,15 @@ defmodule Nerves.Network.EAPoLManager do
 
   ## Examples
 
+  #iex> setup = %{
```

Comment:
This should be an in-file UT, not a doctest.

Reply:
The provided example illustrates the use of the API and that's the goal